docs/porting/display.rst

```diff
@@ -239,6 +239,36 @@ In full and direct modes, the buffer size should be large enough for the whole s
 As LVGL can not handle fractional width make sure to round the horizontal resolution to 8-
 (For example 90 to 96)
 
+The :cpp:func:`lv_draw_sw_i1_convert_to_vtiled` function is used to convert a draw buffer in I1 color format from a row-wise (htiled) to a column-wise (vtiled) buffer layout.
+This conversion is necessary for certain display controllers that require a different draw buffer mapping. The function assumes that the buffer width and height are rounded to a multiple of 8.
+The bit order of the resulting vtiled buffer can be specified using the `bit_order_lsb` parameter.
+For more details, refer to the implementation in :cpp:func:`lv_draw_sw_i1_convert_to_vtiled` in :file:`src/draw/sw/lv_draw_sw.c`.
```

Comment:
@vwheeler63 do we support the `:file:` directive?

Reply:
> @vwheeler63 do we support the `:file:` directive?

Yes.  It is an "interpreted text role" built into Sphinx.  Here is its documentation:

![image](https://github.com/user-attachments/assets/e186c22c-56d8-4b81-9b84-500f27054afe)

Reply:
Thank you!